### DISCUSSION TITLE
Cannot instantiate abstract class error

### DISCUSSION THREAD

**Discussion by rmshub** [pythonnet/pythonnet#2236] [Q&A]
-   Pythonnet version: 3.0.2
-   Python version: 3.11.5
-   Operating System: Windows 10

Observed "Cannot instantiate abstract class error" when instantiating a C# class in python which is derived from an abstract class. Could you please help me to resolve this issue.

**Comment by rmshub** [comment]
Any help on this error ?

**Reply by lostmsu** [reply]
If you think there's a bug in Python.NET you need to provide a reproducible example.
